master/components/dashboard/js/modules/controllers/cAdvisorController.js

```diff
@@ -18,10 +18,12 @@ app.controller('cAdvisorController', [
       $scope.loading = true;
 
       k8sApi.getMinions().success(angular.bind(this, function(res) {
+
         $scope.minions = res;
-        // console.log(res);
-        var promises = lodash.map(res.items, function(m) { return cAdvisorService.getDataForMinion(m.metadata.name); });
 
+        var promises = lodash.map(res.items, function(m) {
+            return cAdvisorService.getDataForMinion(m.metadata.name, (m.status.conditions[0].status === 'True'));
```

Comment:
This is a very bad idea and breaks every cluster where the order of conditions is different. There is `Type` field for that. Look for `Ready` instead of blindly using the first condition in the array.